subsys/bluetooth/mesh/adv_legacy.c

```diff
@@ -94,7 +93,6 @@ static inline void adv_send(struct net_buf *buf)
 	k_sleep(K_MSEC(duration));
 
 	err = bt_le_adv_stop();
-	bt_mesh_adv_send_end(err, cb, cb_data);
```

Comment:
The removal of this line holds the connection state open making the provisioner believe the radio is busy. When reinstated the changes work great and appear to speed up the entire provisioning process.

Simple change: bt_mesh_adv_send_end(err, BT_MESH_ADV(buf));

Reply:
All are postponed to: https://github.com/zephyrproject-rtos/zephyr/blob/7ba52d8f825b62009bb74c553905455b4988dcb0/subsys/bluetooth/mesh/adv.c#L50

Reply:
I see what you are trying to do but it does not close connections cleanly. If I put the implicate end calls back in it works fine.